redis/commands/core.py

```diff
@@ -3255,6 +3256,39 @@ def bzpopmin(self, keys, timeout=0):
         keys.append(timeout)
         return self.execute_command("BZPOPMIN", *keys)
 
+    def bzmpop(
+        self,
+        timeout: float,
+        num_keys: int,
+        keys: List[str],
+        min: Optional[bool] = False,
+        max: Optional[bool] = False,
+        count: Optional[int] = 1,
+    ) -> Optional[list]:
+        """
+        Pop ``count`` values (default 1) off of the first non-empty sorted set
+        named in the ``keys`` list.
+
+        If none of the sorted sets in ``keys`` has a value to pop,
+        then block for ``timeout`` seconds, or until a member gets added
+        to one of the sorted sets.
+
+        If timeout is 0, then block indefinitely.
+
+        For more information check https://redis.io/commands/bzmpop
+        """
+        args = [timeout, num_keys, *keys]
+        if (min and max) or (not min and not max):
+            raise DataError
+        elif min:
+            args.append("MIN")
+        else:
+            args.append("MAX")
+        if count != 1:
```

Comment:
Any reason to not extend either way? We can save the cost of this if check - so it's cheaper. Though, it's unclear if there's a default. What does redis-cli say?

Reply:
In redis-cli if you don't specify the count it will pop 1, so I changhed it to extend either way.